examples/purchaseTesterTypescript/android/settings.gradle

```diff
@@ -6,3 +6,8 @@ project(':react-native-purchases').projectDir = new File(rootProject.projectDir,
 apply from: file("../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesSettingsGradle(settings)
 include ':app'
 includeBuild('../node_modules/react-native-gradle-plugin')
+
+if (file(".composite-enable").exists()) {
+  String path = new File(".composite-enable").text
+  includeBuild (path)
+}
```

Comment:
Can you explain this? Might be useful to leave a comment.

Reply:
Maybe we could also rename the path? `.composite-enable` can be confusing yeah... Maybe `.local-dependency-build-enabled`?